layouts/shortcodes/pinggytunnel.html

```diff
@@ -48,6 +48,39 @@
     }
 }"
 >
+  {{ with .Get "showTabs" }}
+  
+  <div>
+    <ul class="nav nav-tabs" id="myTab" role="tablist">
+      <li class="nav-item" role="presentation">
+          <button class="nav-link active" id="http-tab" data-bs-toggle="tab"
+              data-bs-target="#http-tab-pane" type="button" role="tab" aria-controls="http-tab-pane"
+              aria-selected="true" @click="data.mode = 'http'">HTTP(S)</button>
+      </li>
+      <li class="nav-item" role="presentation">
+          <button class="nav-link" id="tcp-tab" data-bs-toggle="tab" data-bs-target="#http-tab-pane"
+              type="button" role="tab" aria-controls="tcp-tab-pane" aria-selected="false"
+              @click="data.mode = 'tcp'">TCP</button>
+      </li>
+      <li class="nav-item" role="presentation">
+          <button class="nav-link" id="tls-tab" data-bs-toggle="tab" data-bs-target="#http-tab-pane"
+              type="button" role="tab" aria-controls="tls-tab-pane" aria-selected="false"
+              @click="data.mode = 'tls'">TLS</button>
+      </li>
+      <li class="nav-item" role="presentation">
+          <button class="nav-link" id="udp-tab" data-bs-toggle="tab" data-bs-target="#http-tab-pane"
+              type="button" role="tab" aria-controls="udp-tab-pane" aria-selected="false"
+              @click="data.mode = 'udp'">UDP</button>
+      </li>
+  </ul>
+
+  <div class="tab-content" id="advancedSettingsTabContent">
```

Comment:
Do you need tab-content? In any case you are changing content with the help of alpine.